gomod2nix.toml

```diff
@@ -263,8 +263,8 @@ schema = 3
     hash = "sha256-ozwVS2BhAoz+OOisAyMhgg+lq8FdQjf90xoOq9cxtGw="
     replaced = "github.com/crypto-org-chain/go-ethereum"
   [mod."github.com/evmos/ethermint"]
-    version = "v0.6.1-0.20250109075157-62042e2252f6"
-    hash = "sha256-ZmkYPzUBw0pZyoYFckKMq6/yEdtVfWuV7XvEWxycMWg="
+    version = "v0.6.1-0.20250409022716-434f46d2a46c"
```

Comment:
can you also send me the command to update this? 🙏

Reply:
gomod2nix